doc/tech/usecases.md

```diff
@@ -11,3 +11,20 @@
 
 <a name="5.1"></a>
 ## 5.1 Edge
+4-Principle :
+
+This section introduce some principles that should be followed during developing and defining Edge scope for Reference Model , Reference Architecture 01 -OpenStack- , Reference Architecture 02 - K8s – down to Reference implementation and certificate under CNTT
+
+As we are not develop something new , so Edge Principle will follow Reference Model principle “ CNTT Reference Model:Introduction:Principles”  , Reference Architecture principle  “CNTT Reference Architecture : Principles” and Network principle  which in Nutshell 
+
+-	Open source preference
```

Comment:
Maybe some sentences would be good what is the meaning of these from edge point of view.